.github/workflows/build-packages.yml

```diff
@@ -21,20 +22,27 @@ jobs:
         uses: actions/checkout@v2
       - name: Set env variables and install packages
         run: |
-          export BOB_COMMIT_HASH=$(echo ${GITHUB_SHA} | cut -c1-8)
-          export BOB_KEY_SIZE=${{ matrix.bob_key_size }}
+          echo "BOB_COMMIT_HASH=$(echo ${GITHUB_SHA} | cut -c1-8)" >> $GITHUB_ENV
+          echo "BOB_KEY_SIZE=${{ matrix.bob_key_size }}" >> $GITHUB_ENV
           sudo apt-get update && sudo apt-get install -y --no-install-recommends musl-tools
       - name: Install latest nightly
         uses: actions-rs/toolchain@v1
         with:
           toolchain: nightly
           target: ${{ env.TARGET }}
           default: true
+      - name: Print variables
+        run: |
+          echo ${BOB_COMMIT_HASH}
+          echo ${BOB_KEY_SIZE}
+          env
```

Comment:
I suggest not to print all env variables. It is better to print only those you are interested in